zebra-consensus/src/transaction.rs

```diff
@@ -889,11 +938,32 @@ where
 
             // # Consensus
             //
-            // > The Spend transfers and Action transfers of a transaction MUST be
-            // > consistent with its vbalanceSapling value as specified in § 4.13
-            // > ‘Balance and Binding Signature (Sapling)’ on p. 49.
+            // > The Action transfers of a transaction MUST be consistent with
+            // > its v balanceOrchard value as specified in § 4.14.
             //
-            // <https://zips.z.cash/protocol/protocol.pdf#spendsandoutputs>
```

Comment:
This belongs to Sapling and was moved up; the matching Orchard rule was added instead (and it's from https://github.com/ZcashFoundation/zebra/issues/3212). It also should read "Spend transfers and Output transfers", which was already reported.